pkg/kvstore/etcd.go

```diff
@@ -260,7 +259,7 @@ func newEtcdClient(config *client.Config, cfgPath string) (BackendOperations, er
 		firstSession: firstSession,
 		lockPaths:    map[string]*lock.Mutex{},
 		controllers:  controller.NewManager(),
-		RWMutex:      clientMutex,
+		RWMutex:      lock.RWMutex{},
```

Comment:
Wouldn't it work to just remove the field here?

Reply:
Oh really? Let's try it out.